export.go

```diff
@@ -33,7 +37,15 @@ type Exporter struct {
 }
 
 // NewExporter creates a new Exporter. Callers must call Close() when done.
-func newExporter(tree *ImmutableTree) *Exporter {
+func newExporter(tree *ImmutableTree) (*Exporter, error) {
+	if tree == nil {
+		return nil, errors.Wrap(ErrNotInitalizedTree, "tree is nil")
```

Comment:
can we use fmt.Errorf to avoid introducing GitHub.com/pkg/errors as a dependency

Reply:
My mistake.  I thought the team was looking to call Wrap() specifically.  Will change it again.

Reply:
no worries, pkg/errors is deprecated with go 1.18 and beyond because of fmt.errorf and/or errors pkg.

Reply:
Learn something new every day